Speckle_Engine/Convert/ToBHoM.cs

```diff
@@ -8,17 +10,56 @@ namespace BH.Engine.Speckle
 {
   public static partial class Convert
   {
-    /***************************************************/
-    /**** Public Methods                            ****/
-    /***************************************************/
-
-    //Add methods for converting to BHoM from the specific software types, if possible to do without any BHoM calls
-    //Example:
-    //public static Node ToBHoM(this SpeckleNode node)
-    //{
-    //    //Insert code for convertion
-    //}
-
-    /***************************************************/
-  }
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static IEnumerable<IBHoMObject> ResponseToBHoM(ResponseObject response, bool setAssignedId = true, List<string> speckleIds = null)
```

Comment:
Is anything preventing from using the usual convert signature `ToBHoM` (without the prefix) here?

Reply:
Done.